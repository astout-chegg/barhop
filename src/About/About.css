
.image {
    display: block;
    margin-left: auto;
    margin-right: auto;
    width: 75%;
}

.description {
    text-align: center;
}