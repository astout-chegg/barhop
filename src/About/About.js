import React, { Component } from 'react'
import './About.css'

class About extends Component {
    render() {
        return (
            <div className='about'>
                <p>This app was inspired by personal experience having to plan a night out with friends or family
                    and having people not have an idea of where to go first. Utilizing this app will help group of
                    people come together to start the night and knowing at all times where to head next after leaving
                    a bar or restaurant.
                </p>
            </div>
        )
    }
}

export default About;