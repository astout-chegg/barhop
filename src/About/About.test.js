import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import About from './About'

describe(`About component`, () => {
    it('renders the complete about', () => {
        const wrapper = shallow(<About />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})