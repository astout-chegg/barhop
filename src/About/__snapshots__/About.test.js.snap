// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders the complete about 1`] = `
<div
  className="about"
>
  <p>
    This app was inspired by personal experience having to plan a night out with friends or family and having people not have an idea of where to go first. Utilizing this app will help group of people come together to start the night and knowing at all times where to head next after leaving a bar or restaurant.
  </p>
</div>
`;
